<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>197</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">border: 4px solid black;border-radius:10px;</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>7</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <pointsize>21</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: none</string>
              </property>
              <property name="text">
               <string>f(x, y)=</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="function_input">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;border-radius:5px;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="plot_btn">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>17</pointsize>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 3px solid black;border-radius:5px;background-color:#00aa00</string>
              </property>
              <property name="text">
               <string>Plot</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none</string>
                  </property>
                  <property name="text">
                   <string>x:     </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none</string>
                  </property>
                  <property name="text">
                   <string>from</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="x_from_spin_box">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid black;border-radius:4px;</string>
                  </property>
                  <property name="minimum">
                   <double>-9999.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>9999.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>-100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none</string>
                  </property>
                  <property name="text">
                   <string>to</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="x_to_spin_box">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid black;border-radius:4px;</string>
                  </property>
                  <property name="minimum">
                   <double>-9999.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>9999.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none</string>
                  </property>
                  <property name="text">
                   <string>step</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="x_step_spin_box">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid black;border-radius:4px;</string>
                  </property>
                  <property name="maximum">
                   <double>9999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="value">
                   <double>5.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none</string>
                  </property>
                  <property name="text">
                   <string>y:     </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none</string>
                  </property>
                  <property name="text">
                   <string>from</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="y_from_spin_box">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid black;border-radius:4px;</string>
                  </property>
                  <property name="minimum">
                   <double>-9999.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>9999.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>-100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none</string>
                  </property>
                  <property name="text">
                   <string>to</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="y_to_spin_box">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid black;border-radius:4px;</string>
                  </property>
                  <property name="minimum">
                   <double>-9999.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>9999.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="font">
                   <font>
                    <pointsize>13</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none</string>
                  </property>
                  <property name="text">
                   <string>step</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="y_step_spin_box">
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid black;border-radius:4px;</string>
                  </property>
                  <property name="maximum">
                   <double>9999.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                  <property name="value">
                   <double>5.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_21">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none;</string>
                  </property>
                  <property name="text">
                   <string>scale:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="scale_spin_box">
                  <property name="font">
                   <font>
                    <pointsize>17</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 1px solid black;border-radius:5px;</string>
                  </property>
                  <property name="maximum">
                   <double>9999.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="color_btn">
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 2px solid black;border-radius: 5px;background-color: #fff;</string>
                  </property>
                  <property name="text">
                   <string>Color</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="delete_btn">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>15</pointsize>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 3px solid black;border-radius:5px;background-color:#ee0000</string>
                  </property>
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="show_points_checkbox">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none;</string>
                  </property>
                  <property name="text">
                   <string>Show  points</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="show_lines_checkbox">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: none;</string>
                  </property>
                  <property name="text">
                   <string>Show lines</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="error_display_label">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: none;color:red;padding-bottom: 8px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
